include/tenstorrent/bh_chip.h

```diff
@@ -55,12 +55,19 @@ struct bh_chip_data {
 
 	/* notify the main thread to handle therm trip */
 	bool therm_trip_triggered;
+
+	/* notify the main thread to handle pgood events */
+	bool pgood_fall_triggered;
+	bool pgood_rise_triggered;
```

Comment:
Nit- these two should probably be `volatile` since we are writing to them in an ISR. Unlikely to break anything though